scala/src/main/org/apache/spark/api/csharp/CSharpBackend.scala

```diff
@@ -7,15 +7,16 @@ package org.apache.spark.api.csharp
 
 import java.io.{DataOutputStream, File, FileOutputStream, IOException}
 import java.net.{InetAddress, InetSocketAddress, ServerSocket, Socket}
-import java.util.concurrent.{LinkedBlockingQueue, BlockingQueue, TimeUnit}
+import java.util.concurrent.{BlockingQueue, LinkedBlockingQueue, TimeUnit}
 
 import io.netty.bootstrap.ServerBootstrap
 import io.netty.channel.nio.NioEventLoopGroup
 import io.netty.channel.socket.SocketChannel
 import io.netty.channel.socket.nio.NioServerSocketChannel
-import io.netty.channel.{ChannelInitializer, EventLoopGroup, ChannelFuture}
+import io.netty.channel.{ChannelFuture, ChannelInitializer, EventLoopGroup}
 import io.netty.handler.codec.LengthFieldBasedFrameDecoder
 import io.netty.handler.codec.bytes.{ByteArrayDecoder, ByteArrayEncoder}
+import org.apache.spark.internal.Logging
```

Comment:
I'm not sure whether having understood what your mean, if you refer to configure the log level such as from `INFO` to `DEBUG`, you can append `--files log4j.properties` in submit arguments on YARN. Or if you refer to configure different from `$SPARK_CONF/log4j.properties`, we can create a new class, for example `MobiusLogging`, extends `Logging` and override the `initializeLogIfNecessary(isInterpreter: Boolean)` method.